crates/fuel-core/src/executor.rs

```diff
@@ -931,6 +933,10 @@ where
                             block_created,
                         },
                     )?;
+                    // Remove spent coin from owned index
+                    // TODO: avoid updating coin owners index when updating the coin status above
+                    db.storage::<OwnedCoins>()
```

Comment:
I can't find a code where we insert values into the `OwnedCoins` table.

Reply:
Oh, we already had this column. Could you update this code as well, please?

<img width="920" alt="image" src="https://user-images.githubusercontent.com/18346821/223690568-e0145578-8d9d-4d68-892d-2809910202ea.png">